backend/api/Services/MissionSchedulingService.cs

```diff
@@ -210,6 +210,21 @@ public void TriggerRobotAvailable(RobotAvailableEventArgs e)
         {
             OnRobotAvailable(e);
         }
+        private async Task MoveInterruptedMissionsToQueue(IEnumerable<string> interruptedMissionRunIds)
+        {
+            foreach (string missionRunId in interruptedMissionRunIds)
+            {
+                var missionRun = await _missionRunService.ReadById(missionRunId);
+                if (missionRun is null)
+                {
+                    _logger.LogWarning("{Message}", $"Interrupted mission run with Id {missionRunId} could not be found");
+                    continue;
+                }
+
+                missionRun.Status = MissionStatus.Pending;
```

Comment:
Since we have a SetCurrentMissionId in robotService, we could maybe consider having SetCurrentMissionStatus in missionRunService. But if this code already works consistently then this is not needed I suppose.

Reply:
Yes, I agree. Although not added as part of this PR.